components/ws-manager-api/core.proto

```diff
@@ -124,6 +127,29 @@ enum StopWorkspacePolicy {
     ABORT = 2;
 }
 
+enum StopWorkspaceReason {
+    // The user stopped the workspace
+    USER_STOPPED = 0;
+
+    // The user deleted the workspace
+    USER_DELETED = 1;
+
+    // The user cancelled the running prebuild
+    USER_CANCELLED_PREBUILD = 2;
```

Comment:
How is this different from USER_STOPPED? I'd say that if you stop a prebuild, you're effectively doing the same.

Reply:
True, but the code paths are slightly different. Still, I'm more than happy to make `USER_STOPPED` and `USER_CANCELLED_PREBUILD` the same thing.